.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+# PR checks
+
+name: Test
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Deephaven
+        run: |
+          curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/docker-compose.yml > ./test/docker-compose.yml
+          docker-compose -f ./test/docker-compose.yml up -d
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install python dependencies
+        run: |
+            pip install --upgrade pip
+            pip install -r ./assert-tables/requirements.txt
+      - name: Add tables to Deephaven
+        run: python ./test/add-tables.py
```

Comment:
I want to have a (hopefully brief) discussion on this section.

This is very similar (in fact almost identical) to what the action itself does. However, that is only because we're using the python client to initialize the testing Deephaven instance, which just happens to be the same tool that the action uses for the table assertion. Would keeping this here be fine, or should we initialize the Deephaven instance differently (maybe via app mode?) to not confuse anyone looking at this for an example?

We could add something in the README that says something like "Your workflow should build your Deephaven project before using this action", would that be beneficial?